client/src/ui/Form.jsx

```diff
@@ -251,14 +249,11 @@ export default class Form extends Component {
       !Session.get("courseId") ? this.props.course._id : Session.get("courseId"),
       (error, result) => {
         // if (!error && result === 1) {
-        if (error || result.resCode === 1) {
+        if (error || result === 1) {
```

Comment:
Don't you need `result.resCode` (as opposed to `result` for your implementation of success/error messages on the user-side? @AkashAryal

Reply:
Yea that should be `result.resCode`.